dll/win32/setupapi/lang/pt-PT.rc

```diff
@@ -0,0 +1,44 @@
+LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
+
+
+COPYFILEDLGORD DIALOGEX 20, 20, 208, 105
+STYLE DS_MODALFRAME | DS_SHELLFONT | WS_POPUP | WS_VISIBLE | WS_CAPTION
+CAPTION "A copiar ficheiros..."
+FONT 8, "MS Shell Dlg"
+BEGIN
+    PUSHBUTTON "Cancelar", IDCANCEL, 79, 84, 50, 14, WS_CHILD | WS_VISIBLE | WS_TABSTOP
+    LTEXT "Origem:", -1, 7, 7, 77, 11, WS_CHILD | WS_VISIBLE | WS_GROUP
+    LTEXT "", SOURCESTRORD, 7, 18, 194, 11, WS_CHILD | WS_VISIBLE | WS_GROUP
+    LTEXT "Destino:", -1, 7, 30, 77, 11, WS_CHILD | WS_VISIBLE | WS_GROUP
+    LTEXT "", DESTSTRORD, 7, 41, 194, 22, WS_CHILD | WS_VISIBLE | WS_GROUP
+    CONTROL "", PROGRESSORD, "setupx_progress", WS_CHILD | WS_VISIBLE | WS_TABSTOP, 7, 63, 194, 13
+END
+
+IDPROMPTFORDISK DIALOG 0, 0, 260, 120
+STYLE DS_MODALFRAME | WS_CAPTION | WS_SYSMENU
+CAPTION "Arquivos necess�rios"
```

Comment:
You forgot to fix the encoding here (as well as on the lines below).

Reply:
it will be this time